source/Target/Target.cpp

```diff
@@ -2158,17 +2158,17 @@ TypeSystem *Target::GetScratchTypeSystemForLanguage(
                                     .GetFilename()
                                     .AsCString();
             if (m_use_scratch_typesystem_per_module)
-              errs->Printf(
-                  "\nnote: Swift header search options for %s conflict with "
-                  "options found in other modules. Symbols from other modules "
-                  "may not be available.",
-                  module_name);
-            else
+              errs->Printf("\nnote: Swift ClangImporter options for %s "
```

Comment:
Is ClangImporter really something we want to expose? Wouldn't it be better to just say "compiler options"?